ansible/roles/collectory/handlers/main.yml

```diff
@@ -1,10 +1,19 @@
 - name: restart mysql
   service: name={{mysql_service}} state=restarted enabled="yes"
 
+- name: restart collectory
+  service:
+    name: "{{ 'collectory' if exec_jar else tomcat }}"
```

Comment:
should this be `collectory_service_system_user`?

Reply:
Done